marklogic-data-hub-central/ui/src/components/monitor-facet/monitor-facet.tsx

```diff
@@ -21,7 +21,7 @@ interface Props {
 
 const MonitorFacet: React.FC<Props> = (props) => {
   const SHOW_MINIMUM = 3;
-  const SEARCH_MINIMUM = 20;
+  const SEARCH_MINIMUM = 2;
```

Comment:
@federicoRufrancosMarklogic, looks like you changed SEARCH_MINIMUM from 20 to 2 for testing? Can we change it back to 20?

Reply:
ups sorry about that